checkbox-snap/series_classic16/snap/snapcraft.yaml

```diff
@@ -60,3 +60,10 @@ parts:
     plugin: dump
     source: .
     stage: [config_vars]
+  workaround-automated-review-issue:
+    plugin: nil
+    override-build: |
+      # This is a workaround for an upstream bug in the automated snap review tool
+      # remove this once this issue is resolved:
+      # https://bugs.launchpad.net/review-tools/+bug/2049093
+      dd if=/dev/urandom of=$SNAPCRAFT_PART_INSTALL/size_workaround bs=16k count=1
```

Comment:
```suggestion
      yes 'This file is here only to make the snap bigger than 16kB, so the snapcraft properly checksams the snap' | head -c 16384 > $SNAPCRAFT_PART_INSTALL/size_workaround
```

Reply:
I like the idea of the header that states why that file is there, as per the predictable part, I will also explain that in the header. It can't be fully predictable because else the squashfs will collapse it to nothing and it wont save the problem

Reply:
The text I proposed has low entropy, and us such would be easily compressed potentially yielding a very small padding. Do ignore my comment ;)